### ISSUE TITLE
Check/rerun all notebooks before release

### ISSUE THREAD

**Issue by johanvdw** [inbo/niche_vlaanderen#338]

**Comment by johanvdw** [comment]
This was done

**Comment by cecileherr** [comment]
This is not right yet (see https://inbo.github.io/niche_vlaanderen/validation.html#Overview: one module with an error and many modules without results)

**Comment by johanvdw** [comment]
I have rerun them and this is now fixed.

**Comment by johanvdw** [comment]
Keeping this open, I think it would be better if we can run these notebooks in github actions in the future.
